vm/src/obj/objint.rs

```diff
@@ -233,6 +233,12 @@ impl PyInt {
     {
         if cls.is(&vm.ctx.types.int_type) {
             Ok(vm.ctx.new_int(value).downcast().unwrap())
+        } else if cls.is(&vm.ctx.types.bool_type) {
+            Ok(vm
+                .ctx
+                .new_bool(!value.into().eq(&BigInt::zero()))
```

Comment:
there is also `is_zero` method

https://docs.rs/num/0.3.0/num/trait.Zero.html#tymethod.is_zero